truss/tests/conftest.py

```diff
@@ -310,6 +322,15 @@ def no_preprocess_custom_model(tmp_path):
     )
 
 
+@pytest.fixture
+def long_load_model(tmp_path):
+    yield _custom_model_from_code(
+        tmp_path,
+        "long_load_model",
+        NO_PREPROCESS_CUSTOM_MODEL_CODE,
```

Comment:
I think this is the wrong code model variables for this fixture

Reply:
You're right, thats wrong. It should be LONG_LOAD_MODEL_CODE, which it looks like I never switched over.